### ISSUE TITLE
undefined error on typebot-js when importing default

### ISSUE THREAD

**Issue by baptisteArno** [baptisteArno/typebot.io#126]
### Discussed in https://github.com/baptisteArno/typebot.io/discussions/125

<div type='discussions-op-text'>

<sup>Originally posted by **seabysalt** September  5, 2022</sup>
Hey I have no idea why but for some reason, I am struggling to embed my typebot in my next.js application. I am always getting the following error:

<img width="959" alt="Screenshot 2022-09-05 at 16 14 52" src="https://user-images.githubusercontent.com/50774780/188469523-0cb410a3-6cc7-47f8-8f70-b8a2126d6678.png">

However, I am correctly importing Typebot at the top with 
`import Typebot from "typebot-js";`

and I can also see that `"typebot-js": "^2.2.9"` is correctly installed as a dependency in my package.json via npm... Is there something broken? @baptisteArno </div>

**Comment by baptisteArno** [comment]
A workaround is to directly import the function:

```ts
import { initContainer } from 'typebot-js'
```

**Comment by seabysalt** [comment]
thanks! I had actually tried this before as I also thought it could be a solution but for me this wasn't working & still isn't... How come this fix worked for you?

**Comment by baptisteArno** [comment]
> thanks! I had actually tried this before as I also thought it could be a solution but for me this wasn't working & still isn't... How come this fix worked for you?

You get the same error?

**Comment by seabysalt** [comment]
Yes, I do… I tried different things but it looks like it is not detecting
initContainer. I also de-installed and re-installed your package but
nothing working so far… please let me know once you fixed this issue. I was
about to completely switch to your solution for my platform but I would
need this to be working of course and I really cannot figure out why…

Thanks so much in advance!

**Comment by baptisteArno** [comment]
This totally works:

```ts
import { initContainer } from "typebot-js";

const Home: NextPage = () => {
  useEffect(() => {
    initContainer("id", {
      url: "https://viewer.typebot.io/null",
    });
  }, []);
```

Can you paste your code?

**Comment by seabysalt** [comment]
Totally, my bad. I forgot to remove the “Typebot.” before initContainer 🙈
It's working perfectly now. Thanks so much! Then I will now pursue with my
upgrade… Guess, there is no chance I could maybe get like a discount? 😌☺️

**Comment by seabysalt** [comment]
For spotting a bug? 😃
